bluesky/tests/test_zmq.py

```diff
@@ -67,9 +65,9 @@ def local_cb(name, doc):
         local_accumulator.append((name, doc))
 
     # Check that numpy stuff is sanitized by putting some in the start doc.
-    md = {'stuff': {'nested': np.array([1, 2, 3])},
-          'scalar_stuff': np.float64(3),
-          'array_stuff': np.ones((3, 3))}
+    # md = {'stuff': {'nested': np.array([1, 2, 3])},
+    #       'scalar_stuff': np.float64(3),
+    #       'array_stuff': np.ones((3, 3))}
```

Comment:
I think it was intended to be passed to the count plan below. If not, this should be removed along with the comment line above.

Reply:
If the reviewers are sure nobody will ever need this code, it can be removed. The question is if it was disabled temporarily and not enabled back or the change was permanent. That's why I commented it.

Reply:
I see. The comment line suggests it should be used. I wonder if @danielballan or @tacaswell have any memory of it?

Reply:
Yes, it looks like the very next line below---which is also commented out, and has been since this file was first committed---was at some point intended to test that the objects in `md` serialized successfully.

I agree that the fix isn't to remove these lines; it's to use them somewhere. The easiest fix might me to  change the next _uncommented_ line below

```py
RE(count([hw.det]), local_cb)
```

to

```py
RE(count([hw.det]), local_cb, **md)
```